lib/object-generator.js

```diff
@@ -80,6 +81,194 @@ var ObjectGenerator = function () {
           for (var _iterator2 = elements[Symbol.iterator](), _step2; !(_iteratorNormalCompletion2 = (_step2 = _iterator2.next()).done); _iteratorNormalCompletion2 = true) {
             var element = _step2.value;
 
+            object[element.key] = element.generateOriginalObject();
+          }
+        } catch (err) {
+          _didIteratorError2 = true;
```

Comment:
Why don't we switch to `let` and `const` here and remove the `_` and numbers in variables?